coalib/settings/Section.py

```diff
@@ -169,7 +213,7 @@ def __getitem__(self, item, ignore_defaults=False):
         return self.defaults[key]
 
     def __str__(self):
-        value_list = ', '.join(key + ' : ' + repr(str(self.contents[key]))
+        value_list = ', '.join(key + ' : ' + repr(str(self[key]))
```

Comment:
tiny comment: no space before the colon ;) (hey lasse got to comment [this](https://github.com/coala/coala/pull/3178/commits/79852e06cc6ea03e24c068e0bcc7c7328a039c40#r95334456))